frontend/src/components/elements/Spinner/Spinner.test.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react"
+import { shallow } from "src/lib/test_util"
+
+import { Spinner as SpinnerProto } from "src/autogen/proto"
+import Spinner, { SpinnerProps } from "./Spinner"
+
+const getProps = (
+  propOverrides: Partial<SpinnerProps> = {}
+): SpinnerProps => ({
+  element: SpinnerProto.create({
+    text: "Loading...",
+  }),
+  width: 0,
+  ...propOverrides,
+})
+
+describe("Spinner component", () => {
+  it("renders without crashing", () => {
+    const wrapper = shallow(<Spinner {...getProps()} />)
+
+    expect(wrapper.find("Spinner").length).toBe(1)
```

Comment:
nit: would recommend also taking a snapshot of the spinner here

Reply:
Thanks for the suggestion. I added a snapshot test here as well.